src/util/color/rgbcolor.h

```diff
@@ -182,7 +182,11 @@ class RgbColor {
         if (varCode.isNull()) {
             return defaultColor;
         }
+#if QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)
+        VERIFY_OR_DEBUG_ASSERT(varCode.canConvert(QMetaType(QMetaType::UInt))) {
```

Comment:
Using `canConvert<type>()` instead seems to avoid the conditional compilation, e.g. `varCode.canConvert<code_t>()`. This should also work for `DbId`.

Reply:
> Using `canConvert<type>()` instead seems to avoid the conditional compilation, e.g. `varCode.canConvert<code_t>()`. ~~This should also work for `DbId`.~~

Reply:
That is a better solution. Done.